paths_cli/commands/contents.py

```diff
@@ -22,16 +22,16 @@
 
 @click.command(
     'contents',
-    short_help="list named objects from an OPS .nc file",
+    short_help="list named objects from an OPS output file",
 )
 @INPUT_FILE.clicked(required=True)
 @click.option('--table', type=str, required=False,
               help="table to show results from")
 def contents(input_file, table):
-    """List the names of named objects in an OPS .nc file.
+    """List the names of named objects in an OPS output file.
```

Comment:
wouldn't this more suitable to be `storage` file, or just `file`?
```suggestion
    """List the names of named objects in an OPS storage file.
```

```suggestion
    """List the names of named objects in an OPS file.
```